Small update for filename in `isle_integration.md` (#5516)
bytecodealliance/wasmtime: cranelift/docs/isle-integration.md

* `cranelift/codegen/src/prelude.isle`: Common definitions and declarations for
  ISLE. This gets included in every ISLE compilation.

> * `target/.../out/clif_lower.isle`: Auto-generated declarations and helpers for
>   working with CLIF for instruction lowering inside ISLE. Generated by `cranelift/codegen/build.rs`.
>   This gets included in every ISLE compilation.
>   
> * `target/.../out/clif_opt.isle`: Auto-generated declarations and helpers for
>   working with CLIF for mid-end optimizations. Generated by `cranelift/codegen/build.rs`.
>   This gets included in every ISLE compilation.

Reviewer: How about this phrasing?

```suggestion
* `target/.../out/clif_lower.isle`: Auto-generated declarations and helpers
  for working with CLIF for instruction lowering inside ISLE. Generated by
  `cranelift/codegen/build.rs`, which builds it into every backend.
  
* `target/.../out/clif_opt.isle`: Auto-generated declarations and helpers for
  working with CLIF for mid-end optimizations. Generated by
  `cranelift/codegen/build.rs`, which builds it into the mid-end optimizer.
```

Author: Definitely better, thanks!
